target_parquet/helpers.py

```diff
@@ -41,8 +41,6 @@ def flatten(dictionary, flat_schema, parent_key="", sep="__"):
              'key_2__key_3': 2,
              'key_2__key_4__key_5': 3,
              'key_2__key_4__key_6': "['10', '11']"
-             'key_7__key_8': None
-             'key_7__key_9': None
```

Comment:
This inclusion will be done outside the function (when merge with empty record).